SYCL/Matrix/joint_matrix_bf16.cpp

```diff
@@ -0,0 +1,197 @@
+//==-------- joint_matrix_bf16.cpp  - DPC++ joint_matrix--------------- ----==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+// REQUIRES: matrix
```

Comment:
I think we also need some code in `SYCL/lit.cfg.py` to add "marix" to `config.available_features`

Reply:
The change is made in https://github.com/intel/llvm-test-suite/pull/385